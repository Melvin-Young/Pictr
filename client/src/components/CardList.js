import React from 'react';
import Card from './Card';

const CardList = ({ photos }) => (
	<div className="content">
		{photos.map((photoUrl) => {
			return <Card key={photoUrl} src={photoUrl} />;
		})}
	</div>
);

export default CardList;
