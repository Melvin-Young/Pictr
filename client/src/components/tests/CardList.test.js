import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import CardList from '../CardList';
import { photoData } from './Fixtures';

Enzyme.configure({ adapter: new Adapter() });

describe('CardList', () => {
	it('should render', () => {
		const component = shallow(<CardList photos={photoData}/>);
		expect(component).toMatchSnapshot();
	});
});

