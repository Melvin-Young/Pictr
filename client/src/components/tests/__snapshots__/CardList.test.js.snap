// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList should render 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardList
    photos={
      Array [
        Object {
          "height": 148,
          "src": "https://farm5.staticflickr.com/4836/31242683977_ece3744565_m.jpg",
          "width": 240,
        },
        Object {
          "height": 240,
          "src": "https://farm5.staticflickr.com/4839/31242685047_d1c43c2a9c_m.jpg",
          "width": 180,
        },
        Object {
          "height": 240,
          "src": "https://farm5.staticflickr.com/4879/31242685257_91d3cd7ec8_m.jpg",
          "width": 228,
        },
        Object {
          "height": 240,
          "src": "https://farm5.staticflickr.com/4888/31242686317_1ca94e7412_m.jpg",
          "width": 180,
        },
        Object {
          "height": 240,
          "src": "https://farm5.staticflickr.com/4857/32309948418_2c14dab45a_m.jpg",
          "width": 240,
        },
        Object {
          "height": 240,
          "src": "https://farm5.staticflickr.com/4855/32309949888_7bcc666da9_m.jpg",
          "width": 180,
        },
        Object {
          "height": 240,
          "src": "https://farm5.staticflickr.com/4891/32309950258_0661bb338a_m.jpg",
          "width": 180,
        },
        Object {
          "height": 160,
          "src": "https://farm5.staticflickr.com/4888/32309951048_d6586272c0_m.jpg",
          "width": 160,
        },
        Object {
          "height": 160,
          "src": "https://farm5.staticflickr.com/4863/44364705240_98b9eafcca_m.jpg",
          "width": 240,
        },
        Object {
          "height": 180,
          "src": "https://farm5.staticflickr.com/4849/44364706640_d98fdf9190_m.jpg",
          "width": 240,
        },
        Object {
          "height": 148,
          "src": "https://farm5.staticflickr.com/4829/44364707090_8a1974347e_m.jpg",
          "width": 240,
        },
        Object {
          "height": 240,
          "src": "https://farm5.staticflickr.com/4887/45269321345_e27bdd1624_m.jpg",
          "width": 135,
        },
        Object {
          "height": 240,
          "src": "https://farm5.staticflickr.com/4855/45269321735_737d2a2907_m.jpg",
          "width": 180,
        },
        Object {
          "height": 180,
          "src": "https://farm5.staticflickr.com/4898/45457854064_3f27484c3c_m.jpg",
          "width": 240,
        },
        Object {
          "height": 180,
          "src": "https://farm5.staticflickr.com/4851/45457855154_7fc6a2718c_m.jpg",
          "width": 240,
        },
        Object {
          "height": 135,
          "src": "https://farm5.staticflickr.com/4859/45457855494_1e9af3146a_m.jpg",
          "width": 240,
        },
        Object {
          "height": 65,
          "src": "https://farm5.staticflickr.com/4843/46131183302_f8312bfcba_m.jpg",
          "width": 240,
        },
        Object {
          "height": 240,
          "src": "https://farm5.staticflickr.com/4899/46131183632_181394c383_m.jpg",
          "width": 180,
        },
        Object {
          "height": 240,
          "src": "https://farm5.staticflickr.com/4872/46131183732_8aff309656_m.jpg",
          "width": 240,
        },
        Object {
          "height": 135,
          "src": "https://farm5.staticflickr.com/4906/46181991941_df66ce8a14_m.jpg",
          "width": 240,
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Card
          src={
            Object {
              "height": 148,
              "src": "https://farm5.staticflickr.com/4836/31242683977_ece3744565_m.jpg",
              "width": 240,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4839/31242685047_d1c43c2a9c_m.jpg",
              "width": 180,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4879/31242685257_91d3cd7ec8_m.jpg",
              "width": 228,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4888/31242686317_1ca94e7412_m.jpg",
              "width": 180,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4857/32309948418_2c14dab45a_m.jpg",
              "width": 240,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4855/32309949888_7bcc666da9_m.jpg",
              "width": 180,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4891/32309950258_0661bb338a_m.jpg",
              "width": 180,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 160,
              "src": "https://farm5.staticflickr.com/4888/32309951048_d6586272c0_m.jpg",
              "width": 160,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 160,
              "src": "https://farm5.staticflickr.com/4863/44364705240_98b9eafcca_m.jpg",
              "width": 240,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 180,
              "src": "https://farm5.staticflickr.com/4849/44364706640_d98fdf9190_m.jpg",
              "width": 240,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 148,
              "src": "https://farm5.staticflickr.com/4829/44364707090_8a1974347e_m.jpg",
              "width": 240,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4887/45269321345_e27bdd1624_m.jpg",
              "width": 135,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4855/45269321735_737d2a2907_m.jpg",
              "width": 180,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 180,
              "src": "https://farm5.staticflickr.com/4898/45457854064_3f27484c3c_m.jpg",
              "width": 240,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 180,
              "src": "https://farm5.staticflickr.com/4851/45457855154_7fc6a2718c_m.jpg",
              "width": 240,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 135,
              "src": "https://farm5.staticflickr.com/4859/45457855494_1e9af3146a_m.jpg",
              "width": 240,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 65,
              "src": "https://farm5.staticflickr.com/4843/46131183302_f8312bfcba_m.jpg",
              "width": 240,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4899/46131183632_181394c383_m.jpg",
              "width": 180,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4872/46131183732_8aff309656_m.jpg",
              "width": 240,
            }
          }
        />,
        <Card
          src={
            Object {
              "height": 135,
              "src": "https://farm5.staticflickr.com/4906/46181991941_df66ce8a14_m.jpg",
              "width": 240,
            }
          }
        />,
      ],
      "className": "content",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 148,
            "src": "https://farm5.staticflickr.com/4836/31242683977_ece3744565_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 240,
            "src": "https://farm5.staticflickr.com/4839/31242685047_d1c43c2a9c_m.jpg",
            "width": 180,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 240,
            "src": "https://farm5.staticflickr.com/4879/31242685257_91d3cd7ec8_m.jpg",
            "width": 228,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 240,
            "src": "https://farm5.staticflickr.com/4888/31242686317_1ca94e7412_m.jpg",
            "width": 180,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 240,
            "src": "https://farm5.staticflickr.com/4857/32309948418_2c14dab45a_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 240,
            "src": "https://farm5.staticflickr.com/4855/32309949888_7bcc666da9_m.jpg",
            "width": 180,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 240,
            "src": "https://farm5.staticflickr.com/4891/32309950258_0661bb338a_m.jpg",
            "width": 180,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 160,
            "src": "https://farm5.staticflickr.com/4888/32309951048_d6586272c0_m.jpg",
            "width": 160,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 160,
            "src": "https://farm5.staticflickr.com/4863/44364705240_98b9eafcca_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 180,
            "src": "https://farm5.staticflickr.com/4849/44364706640_d98fdf9190_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 148,
            "src": "https://farm5.staticflickr.com/4829/44364707090_8a1974347e_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 240,
            "src": "https://farm5.staticflickr.com/4887/45269321345_e27bdd1624_m.jpg",
            "width": 135,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 240,
            "src": "https://farm5.staticflickr.com/4855/45269321735_737d2a2907_m.jpg",
            "width": 180,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 180,
            "src": "https://farm5.staticflickr.com/4898/45457854064_3f27484c3c_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 180,
            "src": "https://farm5.staticflickr.com/4851/45457855154_7fc6a2718c_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 135,
            "src": "https://farm5.staticflickr.com/4859/45457855494_1e9af3146a_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 65,
            "src": "https://farm5.staticflickr.com/4843/46131183302_f8312bfcba_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 240,
            "src": "https://farm5.staticflickr.com/4899/46131183632_181394c383_m.jpg",
            "width": 180,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 240,
            "src": "https://farm5.staticflickr.com/4872/46131183732_8aff309656_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "[object Object]",
        "nodeType": "function",
        "props": Object {
          "src": Object {
            "height": 135,
            "src": "https://farm5.staticflickr.com/4906/46181991941_df66ce8a14_m.jpg",
            "width": 240,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Card
            src={
              Object {
                "height": 148,
                "src": "https://farm5.staticflickr.com/4836/31242683977_ece3744565_m.jpg",
                "width": 240,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 240,
                "src": "https://farm5.staticflickr.com/4839/31242685047_d1c43c2a9c_m.jpg",
                "width": 180,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 240,
                "src": "https://farm5.staticflickr.com/4879/31242685257_91d3cd7ec8_m.jpg",
                "width": 228,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 240,
                "src": "https://farm5.staticflickr.com/4888/31242686317_1ca94e7412_m.jpg",
                "width": 180,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 240,
                "src": "https://farm5.staticflickr.com/4857/32309948418_2c14dab45a_m.jpg",
                "width": 240,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 240,
                "src": "https://farm5.staticflickr.com/4855/32309949888_7bcc666da9_m.jpg",
                "width": 180,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 240,
                "src": "https://farm5.staticflickr.com/4891/32309950258_0661bb338a_m.jpg",
                "width": 180,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 160,
                "src": "https://farm5.staticflickr.com/4888/32309951048_d6586272c0_m.jpg",
                "width": 160,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 160,
                "src": "https://farm5.staticflickr.com/4863/44364705240_98b9eafcca_m.jpg",
                "width": 240,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 180,
                "src": "https://farm5.staticflickr.com/4849/44364706640_d98fdf9190_m.jpg",
                "width": 240,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 148,
                "src": "https://farm5.staticflickr.com/4829/44364707090_8a1974347e_m.jpg",
                "width": 240,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 240,
                "src": "https://farm5.staticflickr.com/4887/45269321345_e27bdd1624_m.jpg",
                "width": 135,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 240,
                "src": "https://farm5.staticflickr.com/4855/45269321735_737d2a2907_m.jpg",
                "width": 180,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 180,
                "src": "https://farm5.staticflickr.com/4898/45457854064_3f27484c3c_m.jpg",
                "width": 240,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 180,
                "src": "https://farm5.staticflickr.com/4851/45457855154_7fc6a2718c_m.jpg",
                "width": 240,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 135,
                "src": "https://farm5.staticflickr.com/4859/45457855494_1e9af3146a_m.jpg",
                "width": 240,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 65,
                "src": "https://farm5.staticflickr.com/4843/46131183302_f8312bfcba_m.jpg",
                "width": 240,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 240,
                "src": "https://farm5.staticflickr.com/4899/46131183632_181394c383_m.jpg",
                "width": 180,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 240,
                "src": "https://farm5.staticflickr.com/4872/46131183732_8aff309656_m.jpg",
                "width": 240,
              }
            }
          />,
          <Card
            src={
              Object {
                "height": 135,
                "src": "https://farm5.staticflickr.com/4906/46181991941_df66ce8a14_m.jpg",
                "width": 240,
              }
            }
          />,
        ],
        "className": "content",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 148,
              "src": "https://farm5.staticflickr.com/4836/31242683977_ece3744565_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4839/31242685047_d1c43c2a9c_m.jpg",
              "width": 180,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4879/31242685257_91d3cd7ec8_m.jpg",
              "width": 228,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4888/31242686317_1ca94e7412_m.jpg",
              "width": 180,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4857/32309948418_2c14dab45a_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4855/32309949888_7bcc666da9_m.jpg",
              "width": 180,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4891/32309950258_0661bb338a_m.jpg",
              "width": 180,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 160,
              "src": "https://farm5.staticflickr.com/4888/32309951048_d6586272c0_m.jpg",
              "width": 160,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 160,
              "src": "https://farm5.staticflickr.com/4863/44364705240_98b9eafcca_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 180,
              "src": "https://farm5.staticflickr.com/4849/44364706640_d98fdf9190_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 148,
              "src": "https://farm5.staticflickr.com/4829/44364707090_8a1974347e_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4887/45269321345_e27bdd1624_m.jpg",
              "width": 135,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4855/45269321735_737d2a2907_m.jpg",
              "width": 180,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 180,
              "src": "https://farm5.staticflickr.com/4898/45457854064_3f27484c3c_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 180,
              "src": "https://farm5.staticflickr.com/4851/45457855154_7fc6a2718c_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 135,
              "src": "https://farm5.staticflickr.com/4859/45457855494_1e9af3146a_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 65,
              "src": "https://farm5.staticflickr.com/4843/46131183302_f8312bfcba_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4899/46131183632_181394c383_m.jpg",
              "width": 180,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 240,
              "src": "https://farm5.staticflickr.com/4872/46131183732_8aff309656_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "[object Object]",
          "nodeType": "function",
          "props": Object {
            "src": Object {
              "height": 135,
              "src": "https://farm5.staticflickr.com/4906/46181991941_df66ce8a14_m.jpg",
              "width": 240,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
